contrib/sssd.spec.in

```diff
@@ -36,6 +36,12 @@
 %global build_passkey 0
 %endif
 
+%if 0%{?rhel} >= 10
```

Comment:
Since this depends on sssd user, it would be good to add comment and move it to line 9 where we define sssd user.

Reply:
Fixed, thanks.